src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -227,6 +229,22 @@
             <argument type="service" id="api_platform.resource_class_resolver" />
         </service>
 
+        <service id="api_platform.identifier_normalizer" class="ApiPlatform\Core\Identifier\Normalizer\ChainIdentifierNormalizer" public="false">
```

Comment:
`api_platform.identifier.normalizer` for consistency with the other services that have been introduced?